src/core/lib/transport/metadata.h

```diff
@@ -277,8 +277,8 @@ class RefcountedMdBase {
   /* must be byte compatible with grpc_mdelem_data */
   grpc_slice key_;
   grpc_slice value_;
-  grpc_core::Atomic<intptr_t> refcnt_;
```

Comment:
is this a cache line optimization?

Reply:
No, this is to make the layout of this and the static md object match so that there's one less branch inside hpack_encoder.cc (where we lookup the hash for the metadata, which is either static or interned (which would use this struct)).